easybuild/easyconfigs/g/GC3Pie/GC3Pie-2.4.1.eb

```diff
@@ -0,0 +1,111 @@
+easyblock = 'Bundle'
+
+name = 'GC3Pie'
+version = '2.4.1'
+
+homepage = 'https://gc3pie.readthedocs.org'
+description = """GC3Pie is a Python package for running large job campaigns on diverse batch-oriented execution
+ environments."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+# this is a bundle of Python packages
+exts_defaultclass = 'PythonPackage'
+exts_filter = ('python -c"import %(ext_name)s"', '')
+
+# allow use of system Python
+allow_system_deps = [('Python', SYS_PYTHON_VERSION)]
+
+exts_list = [
+    ('setuptools', '18.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/s/setuptools/'],
+    }),
+    # required for paramiko
+    ('ecdsa', '0.13', {
+        'source_urls': ['https://pypi.python.org/packages/source/e/ecdsa/'],
+    }),
+    # required for paramiko
+    ('pycrypto', '2.6.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/pycrypto/'],
+        'modulename': 'Crypto',
+    }),
+    ('paramiko', '1.15.2', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/paramiko/'],
+    }),
+    ('PrettyTable', '0.7.2', {
+        'source_urls': ['https://pypi.python.org/packages/source/P/PrettyTable/'],
+        'source_tmpl': 'prettytable-%(version)s.tar.gz',
+    }),
+    ('pyCLI', 'devel', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/pyCLI/'],
+        'modulename': 'cli',
+    }),
+    ('SQLAlchemy', '1.0.8', {
+        'source_urls': ['https://pypi.python.org/packages/source/S/SQLAlchemy/'],
+    }),
+    ('parsedatetime', '1.5', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/parsedatetime/'],
+    }),
+    ('boto', '2.38.0', {
```

Comment:
packages at lines 49--87 are only required for cloud support, which -I guess- is not that interesting for EasyBuild users[^1].  They can be safely removed.

[^1]: GC3Pie cloud backends spawn a short-lived VM and kill it when the job is over.  For EasyBuild, this means that you'd lose the results as soon as the job is over.